fe/fe-core/src/main/java/com/starrocks/clone/TabletScheduler.java

```diff
@@ -910,7 +913,9 @@ private void handleReplicaMissing(TabletSchedCtx tabletCtx, AgentBatchTask batch
         Preconditions.checkNotNull(destPath);
         tabletCtx.setDest(destPath.getBeId(), destPath.getPathHash());
 
-        if (Config.recover_with_empty_tablet && tabletCtx.getReplicas().size() == 1) {
+        if (Config.recover_with_empty_tablet
```

Comment:
Consider a situation: tablet has single replica(replication_num=1), and that replica is corrupted, user want to recover it with an empty tablet, by setting it as bad.
With this condition, can we achieve that?

Reply:
Add isBad check in isDataLost()